### ISSUE TITLE
More context and better snippets for `cookbook/jettons`

### ISSUE THREAD

**Issue by anton-trunov** [tact-lang/tact-docs#399]
This is a follow-up issue after #321.

Snippets requirements: 
- Snippets showing recipes MUST (in the sense of RFC 2119) compile, for instance the snippet in "Sending jetton transfer" is not valid Tact and needs to be manually augmented to compile
- All recipes MUST be self-contained and have a (simple) purpose, for instance, we need to replace `// your logic of processing token notification` with some exact logic
- "myJettonWalletAddress may also be stored in contract storage to use less gas in every transaction" -- we should refactor the examples to use this approach
- "forwardTonAmount: 1" isn't it too much Toncoins?

We should expand the explanations by adding more context aiming at novice TON developers.
- Explain what are Jettons and what structure they have
- What is "Transfer notification message", this should explain the "YOUR contract's jetton wallet"
- Explain why "Sender of transfer notification must be validated!"
- "Wallet initial data layout is shown below, but sometimes it can differ" -- we need to at least briefly mention the other cases

**Comment by anton-trunov** [comment]
@a-bahdanau Could you please help us with this follow-up?

**Comment by a-bahdanau** [comment]
> @a-bahdanau Could you please help us with this follow-up?

Yep, this is on me

**Comment by anton-trunov** [comment]
Awesome!

**Comment by a-bahdanau** [comment]
@anton-trunov Hello! 

> * Snippets showing recipes MUST

Do you mean that all code should be placed in one block of code? Or it can be devided into separate parts to avoid dublication? Like with `JettonTransferNotification`

**Comment by anton-trunov** [comment]
Hey @a-bahdanau, the cookbook format usually implies every snippet to be self-contained. Would be nice if Tact users could just copy-paste examples and have something immediately compilable
